src/Domain/Enum/TelegramUpdateObjectsList.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Enum;
+
+enum TelegramUpdateObjectsList: string
```

Comment:
Предлагаю сделать суффикс 'Enum', поскольку списком также может быть, например, коллекция объектов.

```suggestion
enum TelegramUpdateObjectsEnum: string
```